test_data/issue-161.bad.nix

```
@@ -0,0 +1,17 @@
+(
+  builtins.foldl' (
+    acc: v: let
+      isOperator = builtins.typeOf v == "list";
+      operator = if isOperator then (builtins.elemAt v 0) else acc.operator;
+    in
+      if isOperator then (acc // { inherit operator; }) else {
+        inherit operator;
+        state = operators."${operator}" acc.state (satisfiesSemver pythonVersion v);
+      }
+  )
+    {
+      operator = ",";
+      state = true;
+    }
+    tokens
+).state;
```

```suggestion
).state
```
removed the semicolon since it's not valid syntax